### ISSUE TITLE
Commercial: Friendly Captcha element is not highlighted when trying to submit form prior to completing the captcha

### ISSUE THREAD

**Issue by makuhlmann** [shopware/platform#15407]
### Shopware Version

Cloud, 6.7.8.0 w/ SwagCommercial 7.8.0

### Affected area / extension

Extension:Commercial

### Actual behaviour

When the Friendly Captcha has not been completed prior to sending a form, it will not be highlighted anymore as other missing mandatory fields would be. This behaviour differs from the one previously in Shopware 6.6, where the captcha is highlighted with a red box when it is still to be solved.

<details><summary>Shopware 6.6</summary>
<p>

<img width="1201" height="1226" alt="Image" src="https://github.com/user-attachments/assets/82abb880-69a8-4580-b526-b9004644036a" />

</p>
</details> 

<details><summary>Shopware 6.7 & Cloud</summary>
<p>

<img width="1156" height="1360" alt="Image" src="https://github.com/user-attachments/assets/a6dd8773-841f-4de2-9551-073403e3e85d" />

</p>
</details> 

This behaviour may confuse less attentive or tech literate customers, or customers using accessibility tools, since there is no indication or error whatsoever as to what is missing or why the form would not submit.

reCAPTCHA is not affected either, a red error message shows up when it has not been completed yet.

### Expected behaviour

The captcha would be highlighted as to be completed

### How to reproduce

- Set up a new shop and install SwagCommercial
- Enable Friendly Captcha using dummy data, no actual data needed for this test
- Open the registration page and click the submit button without filling anything in
